modules/panels.nix

```diff
@@ -155,6 +157,13 @@ let
     (cfg.workspace.wallpaperPictureOfTheDay != null) ||
     (cfg.workspace.wallpaperPlainColor != null) ||
     ((builtins.length cfg.panels) > 0));
+
+  anyDesktopFolderSettingsSet = (
+    (cfg.workspace.desktop.icons.arrangement != null) ||
+    (cfg.workspace.desktop.icons.size != null) ||
+    (cfg.workspace.desktop.icons.folderPreviewPopups != null) ||
+    (cfg.workspace.desktop.icons.previewPlugins != null)
+  );
```

Comment:
Here I believe `lib.any (v: v != null) (builtins.attrValues cfg.workspace.desktop.icons)` would be more elegant.

Reply:
Definitely, thank you!